python/generate_schema.py

```diff
@@ -2,9 +2,8 @@
 import json
 
 from fastapi.openapi.utils import get_openapi
-from starlette.routing import Mount
-
 from sidecar.typing import CliCommands
+from starlette.routing import Mount
```

Comment:
This is more a question for @gjreda and #26 but I've had good luck using [isort](https://pycqa.github.io/isort/) to enforce a sort order for imports in Python in the past. The usual grouping is: standard library imports, third-party imports, your own imports.

Reply:
@danvk I'm a fan of that pattern and thought I'd set it up as part of #419, but maybe I missed a piece.